nvfuser/pytorch_utils.py

```diff
@@ -54,3 +56,124 @@ def patch_codegen_so():
         os.path.join(src_dir, "libnvfuser_codegen.so"),
         os.path.join(dst_dir, "libnvfuser_codegen.so"),
     )
+
+
+def get_device_properties() -> Tuple[int, float]:
```

Comment:
I don't know how heavy this function is. If we will call `clear_cuda_cache` many times, shall we add a `@functools.lru_cache(None)` to this function?

Reply:
That's an execellent point. It will be called a high number of times -- which is why I initially had it as a global variable computed once.
Where can I see the benchmark wall time -- that can give us an estimate how huge the overhead is.